braph2genesis/graphs/_OrdMlWU.gen.m

```diff
@@ -0,0 +1,259 @@
+The layers are connected in an ordinal fashion.
+
+%% ¡props_update!
+
+%%% ¡prop!
+
+NAME (constant, string) is the name of the ordinal multilayer weighted undirected graph.
+%%%% ¡default!
+'OrdMlWU'
+
+%%% ¡prop!
+DESCRIPTION (constant, string) is the description of the ordinal multilayer weighted undirected graph.
+%%%% ¡default!
+'In an ordinal multilayer weighted undirected (WU) graph, layers could have different number of nodes with within-layer weighted undirected edges, associated with a real number between 0 and 1 and indicating the strength of the connection. The connectivity matrices are symmetric. There are connections between layers connecting the corresponding nodes.The layers are connected in an ordinal fashion.'
+
+%%% ¡prop!
+TEMPLATE (parameter, item) is the template of the ordinal multilayer weighted undirected graph.
+%%% ¡_prop!
+% % % TEMPLATE (parameter, item) is the graph template to set the graph and measure parameters.
+% % % %%%% ¡_settings!
+% % % 'OrdMlWU'
+
+%%% ¡prop!
+ID (data, string) is a few-letter code for the ordinal multilayer weighted undirected graph.
+%%%% ¡default!
+'OrdMlWU ID'
+
+%%% ¡prop!
+LABEL (metadata, string) is an extended label of the ordinal multilayer weighted undirected graph.
+%%%% ¡default!
+'OrdMlWU label'
+
+%%% ¡prop!
+NOTES (metadata, string) are some specific notes about the ordinal multilayer weighted undirected graph.
+%%%% ¡default!
+'OrdMlWU notes'
+
+%%% ¡prop!
+GRAPH_TYPE (constant, scalar) returns the graph type __Graph.ORDERED_MULTILAYER__.
+%%%% ¡default!
+Graph.ORDERED_MULTILAYER
+
+%%% ¡prop!
+CONNECTIVITY_TYPE (query, smatrix) returns the connectivity type __Graph.WEIGHTED__ * ones(layernumber).
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.WEIGHTED * ones(layernumber);
+
+%%% ¡prop!
+DIRECTIONALITY_TYPE (query, smatrix) returns the directionality type __Graph.UNDIRECTED__ * ones(layernumber).
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.UNDIRECTED * ones(layernumber);
+
+%%% ¡prop!
+SELFCONNECTIVITY_TYPE (query, smatrix) returns the self-connectivity type __Graph.NONSELFCONNECTED__ on the diagonal and __Graph.SELFCONNECTED__ off diagonal.
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.SELFCONNECTED * ones(layernumber);
+value(1:layernumber+1:end) = Graph.NONSELFCONNECTED;                
+
+%%% ¡prop!
+NEGATIVITY_TYPE (query, smatrix) returns the negativity type __Graph.NONNEGATIVE__ * ones(layernumber).
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.NONNEGATIVE * ones(layernumber);
+
+%%% ¡prop!
+A (result, cell) is the cell containing the multilayer weighted adjacency matrices of the multilayer weighted undirected graph.
```

Comment:
it contains more than adjacency matrices. So either you say that it contains the **multilayer weighted supra-adjacency matrix** or you say that it contains the within-layer weighted adjacency matrices of the multilayer weighted undirected graph and the ordinal connections between layers.